zubhub_frontend/zubhub/src/components/breadCrumb/breadCrumb.jsx

```diff
@@ -61,8 +60,8 @@ function BreadCrumb() {
   };
 
   const createPathList = () => {
-    let pathList = match.path.split('/');
-    let pathListUrl = match.url.split('/');
+    let pathList = window.location.pathname.split('/');
```

Comment:
was `match.patch.split` causing any issues after the upgrade?

Reply:
react-router-dom 6